circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+  services:
+    - docker
+
+test:
+  override:
+    - echo 0
+
+database:
```

Comment:
Probably worth adding a comment above `database:` that we generally include here:
```
# Use this section to setup tests external dependencies, not just databases.
```